z/zfs.yml

```diff
@@ -12,7 +12,6 @@ zfs:
 #  - /usr/bin/ros:/usr/bin/ros
   volumes_from:
   - all-volumes
-  restart: always
```

Comment:
I was making it restart so that we re-test that things are installed - that way when someone upgrades the kernel, or changes the console, it is detected, and dealt with.

otherwise, those types of system changes just break zfs and they have to do it again.